src/mmds/src/lib.rs

```diff
@@ -65,6 +65,20 @@ fn build_response(http_version: Version, status_code: StatusCode, body: Body) ->
     response
 }
 
+// Make the URI a correct JSON pointer value.
+fn sanitize_uri(uri: &str) -> String {
+    let mut sanitized_uri = uri.to_owned();
+    let mut len = u32::MAX as usize;
+    // Loop while the deduping decreases the sanitized len.
```

Comment:
```suggestion
    // Loop until the sanitized len is no longer modified in an iteration. 
```

Reply:
I think my initial comment has same meaning, why do you prefer it to be longer ?

Reply:
For me the comment wasn't clear about when does the looping end and it seemed that this was its purpose. From the comment I'm thinking of an infinite loop. Anyway, it was more of a nit :D.